security/oc_tls.c

```diff
@@ -1368,6 +1435,11 @@ static int
 oc_tls_load_identity_cert_chain(mbedtls_ssl_config *conf, size_t device,
                                 int credid)
 {
+  if (credid < OC_TLS_SELECTED_ANY_CRED_ID) {
```

Comment:
I see, we use this feature in DPS, where we use `-2` to skip identity certificates and force mfg certificates. We should document it in some relevant header, because it might get removed if it is not mentioned anywhere.